packages/itmat-ui-react/src/components/users/userDetails.tsx

```diff
@@ -81,6 +81,9 @@ export const EditUserForm: React.FunctionComponent<{ user: (IUserWithoutToken &
                     <label>Description:  <input type='text' value={inputs.description} onChange={e => { setInputs({ ...inputs, description: e.target.value }) }} /></label> <br /><br />
                     <label>Organisation: <input type='text' value={inputs.organisation} onChange={e => setInputs({ ...inputs, organisation: e.target.value })} /> </label><br /><br />
                     <label>Created by (readonly): <input type='text' readOnly value={inputs.createdBy} /> </label><br /><br />
+                    <label>Created at (readOnly): <input type='text' readOnly value={(new Date(inputs.createdAt)).toLocaleString()} /></label><br /><br />
+                    <label>Expired at: <input type='datetime-local' value={(new Date(inputs.expiredAt)).toISOString().substring(0, 16) } onChange={e => { setInputs({ ...inputs, expiredAt: (new Date(e.target.value)).getTime() }) }} /></label><br /><br />
```

Comment:
just a thought - is it better to have:
1. uniformly a expiry window "say 2 months", after which admin has to revive; or
2. let admin select a date (e.g. 20-7-2020) explicitly; or
3. let admin select a relative date (e.g after 20 days) explicitly

open to suggstion

Reply:
Sure, would complete that, thanks!!!

Reply:
no i mean, we could pick one of these options

Reply:
@fguitton thoughts